src/azure-cli/azure/cli/command_modules/vm/_template_builder.py

```diff
@@ -626,8 +629,14 @@ def _build_storage_profile():
     if encryption_at_host is not None:
         vm_properties['securityProfile']['encryptionAtHost'] = encryption_at_host
 
-    if security_type is not None:
+    # The `Standard` is used for backward compatibility to allow customers to keep their current behavior
+    # after changing the default values to Trusted Launch VMs in the future.
+    if security_type is not None and security_type != "Standard":
         vm_properties['securityProfile']['securityType'] = security_type
+    else:
+        logger.warning('Consider upgrading security for your workloads using Azure Trusted Launch VMs. '
+                       'To know more about Trusted Launch, please visit '
+                       'https://learn.microsoft.com/en-us/azure/virtual-machines/trusted-launch.')
```

Comment:
```suggestion
                       'https://aka.ms/TLaD.')
```
We can use the short link uniformly to make notification message more concise. What do you think of it?

Reply:
@zhoxing-ms - The short link will be moved to Trusted Launch as default blog post once it's published which will not be relevant to end users who are choosing to not use Trusted Launch configuration. Hence pointing to learn docs to know more about benefits of TL.